pkg/reconciler/ingress/ingress.go

```diff
@@ -270,7 +270,7 @@ func (r *BaseIngressReconciler) reconcileIngress(ctx context.Context, ra Reconci
 	ia.GetStatus().MarkLoadBalancerReady(lbs, publicLbs, privateLbs)
 	ia.GetStatus().ObservedGeneration = ia.GetGeneration()
 
-	if enablesAutoTLS(ctx) {
+	if enableReconcileGateway(ctx) {
```

Comment:
Would it be a future PR to have a way to make use of this flag without AutoTLS enabled? As far as I understand, even with this flag, Gateway will be reconciled using tls information on Route which will only get populated when the AutoTLS flag is set.

Reply:
Is it also correct to assume that there will be some step in between to reference an existing cert instead of calling the [MakeCertificates() function](https://github.com/knative/serving/blob/master/pkg/reconciler/route/route.go#L309) in future PRs?
Also allow to choose mode of TLS as [MakeServers()](https://github.com/knative/serving/blob/master/pkg/reconciler/ingress/resources/gateway.go#L88) only allow simple TLS for now?

Reply:
Yes. This PR is just a first step of the manual TLS feature request. There will be a future PR in the Route controller to consume manual provisioned Certificates.